app/assets/javascripts/discourse/app/templates/user/messages.hbs

```diff
@@ -32,13 +32,13 @@
         {{/link-to}}
       </li>
       <li class="noGlyph">
```

Comment:
I know this is not your pr, but this is an invalid class name we would prefer no-glyph, no need to fix it there though

Reply:
I made a separate PR in https://github.com/discourse/discourse/pull/14100